api/services/expenditureService.ts

```diff
@@ -163,6 +163,7 @@ export async function getExpendituresAsync(expendituresAttrs: IGetExpenditureAtt
         }
         return expenditures;
     } catch (e) {
+        console.log('getExpendituresAsync', e);
```

Comment:
Is this something that will stay put or is it temporary?

Reply:
I'm going to cut it now! Was helpful in debugging, but not a useful console log like the others.